gusto/solvers/linear_solvers.py

```diff
@@ -52,6 +58,23 @@ def __init__(self, equations, alpha=0.5, solver_parameters=None,
         self.dt = equations.domain.dt
         self.alpha = alpha
 
+        # Set relaxation parameters. If an alternative has not been given, set
+        # to semi-implicit off-centering factor
+        if tau_u is not None:
```

Comment:
these could actually be fit on one line nicely, e.g.
`self.tau_u = tau_u if tau_u is not None else alpha`